src/components/home/Home.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useState } from 'react';
-import { Button, Input } from 'antd';
 import { Link } from 'react-router-dom';
 import { RightOutlined } from '@ant-design/icons';
+import { Button, Input } from 'antd'; // Add this import
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import {
```

Comment:
- [ OPTIONAL] According to the project requirements:
> The App should have unit tests for several components of the app and some pure functions in Redux code.
The App should have integration tests.

Kindly make sure to add more unit tests and integration tests using React Testing Librar. ✍🏻